libmscore/box.h

```diff
@@ -110,6 +110,7 @@ class HBox final : public Box {
       HBox* clone() const override      { return new HBox(*this); }
       ElementType type() const override { return ElementType::HBOX; }
 
+      Fraction tick() const;
```

Comment:
This is an override of `Element::tick()`, isn't it? So it should be declared as:

    Fraction tick() const override;

I noticed that the declaration of `MeasureBase::tick()` is also missing the same `override` specifier. Maybe throw that one in too?

Reply:
Indeed, that's what I copied/pasted from :-).  Yes, I will add the override to both.